### PR TITLE
Add language codes

### LINKED ISSUE #55: Add new API Property - language_codes
Short discussion with @jav7zaid (#53) brought me to idea that we could deliver a `language_codes` Property. `language_codes` Property is an `Array` of `String` - there are countries that have multiple languages. It would let _API consumers_ format their data with specific country format or setting `lang` attribute for _HTML_ Tags.

To solve that, we can use combination of _ISO 639-1_ and _ISO 3166-1 alpha-2_ codes (e.g.: `pl-PL`).
Sample `Country {Object}`:
```js
{
  "country": "poland",
  "capital": "warsaw",
  "currency": "zloty",
  "native_language": ["polish"],
  "famous_for": "pierogi and potatoes",
  "phone_code": "+48",
  "flag": "https://flagpedia.net/data/flags/h80/pl.png",
  "drive_direction": "right",
  // NEW DATA
  "language_codes": ["pl-PL"]
}
```

Let me know what you think! ✌️

### PR DESCRIPTION
**Pull request by anneloreegger**
Fixes #55

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -15,7 +15,8 @@
       "alpha_3": "afg"
     },
     "tld": ".af",
-    "constitutional_form": "republic"
+    "constitutional_form": "republic",
+    "language_codes": ["ps-AF", "fa-AF"]
```
Comment (sthiepaan) [inline data/data.json:19]: Can we rearrange `language_codes {Array}` elements to be sorted in alphabetical order? 
(same for lines: 19, 323, 399, 532, 570 628, 647, 1065, 1464, 1540, 1559, 1673, 1749, 1863, 1958, 1996, 2091, 2110, 2338, 2357, 2528, 2642, 2752, 2824, 2932, 3058, 3094, 3256, 3274, 3328, 3400, 3436, 3454, 3544,  3580,

File: data/data.json [hunk 2]
```diff
@@ -2535,8 +2676,8 @@
       "alpha_3": "prt"
     },
     "tld": ".pt",
-    "constitutional_form": "republic"
-  },
+    "constitutional_form": "republic",
+    "language_codes": ["pt-PT"]},
```
Comment (sthiepaan) [inline data/data.json:2680]:
```suggestion
    "language_codes": ["pt-PT"]
  },
```
Can we fix the formatting? (same for lines: 2680, 2698, 2716, 2734, 2752, 2770, 2788, 2806, 2824, 2842, 2860, 2878, 2896, 2914, 2932, 2950, 2968, 2986, 3004, 3022, 3040, 3058, 3076, 3094, 3112, 3130, 3148, 3166, 3202, 3220, 3238, 3256, 3274, 3292, 3310, 3328, 3346, 3364, 3382, 3400, 3418, 3436, 3454, 3472, 3490, 3508, 3526, 3544, 3562, 3580, 3598, 3616, 3634, 3652, 3670)

**Review by sthiepaan — CHANGES_REQUESTED** [review]
Nice job @anneloreegger! I've checked your work and it looks promising!

**Comment by sthiepaan** [comment]
@anneloreegger please remember to run `npm test` to check it existing methods are still working after your changes 😉

**Comment by anneloreegger** [comment]
@sthiepaan Thanks for looking at my pull request so quickly. I already implemented the requested changes, and also adapted the tests ;)

**Review by sthiepaan — APPROVED** [review]
LGTM 🎉

**Pull request merged**